gpkitmodels/GP/aircraft/wing/wing_test.py

```diff
@@ -39,9 +39,8 @@ def wing_test():
     from gpkit import settings
     if settings["default_solver"] == "cvxopt":
         for l in loading:
-            for v in ["\\bar{M}_{tip}", "\\bar{S}_{tip}",
-                      "\\bar{\\delta}_{root}", "\\theta_{root}"]:
-                l.substitutions[v] = 1e-3
+            for v in ["Mtip", "Stip", "wroot", "throot"]:
+                l.substitutions[v] = 1e-1
```

Comment:
huh why so much larger?

Reply:
I'm not sure.  But it gave me unknown at 1e-2 so...